src/sql/workbench/services/accountManagement/browser/accountDialog.ts

```diff
@@ -209,6 +212,9 @@ export class AccountDialog extends Modal {
 
 	protected renderBody(container: HTMLElement) {
 		this._container = container;
+		// Setup loading spinner
+		this._loadingSpinner = new LoadingSpinner(this._container, { showText: true });
```

Comment:
I saw that you are setting the loading text multiple times in below, you should set the loading text, loading completed text here just once, and after that all you need to manipulate is the `loading` property.

Reply:
so you use these for multiple purposes? loading accounts and adding accounts?

Reply:
The adding account function used the Modal spinner to display the loading status, I decided to upgrade it to use the new spinner for consistency.